ast/src/parsed/display.rs

```diff
@@ -681,6 +685,55 @@ impl<E: Display> Display for EnumVariant<E> {
     }
 }
 
+impl<E: Display> Display for StructDeclaration<E> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
+        let name = self.name.to_string();
+        let type_vars = if self.type_vars.is_empty() {
```

Comment:
Maybe we could create a helper for this, I have seen it a lot. The same code is also inside `TypeScheme::type_vars_to_string`